docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: '2'
+services:
+  BANZAIpipelineE2EData:
+    image: docker.lco.global/banzai-e2e-data:1.0.1
+    entrypoint:
+    - /bin/true
+    labels:
+      io.rancher.container.pull_image: always
+      io.rancher.container.start_once: 'true'
+    volumes:
+    - banzaie2e:/archive
+    logging:
+      options:
+        max-size: '100m'
+        max-file: '3'
+  AstrometryCatalogs:
+    image: docker.lco.global/astrometry.net-catalogs:1.0.0
+    entrypoint:
+    - /bin/true
+    labels:
+      io.rancher.container.pull_image: always
+      io.rancher.container.start_once: 'true'
+    volumes:
+    - astrometry:/opt/astrometry.net/data
+    logging:
+      options:
+        max-size: '100m'
+        max-file: '3'
+  BANZAITest:
+    image: ${DOCKER_IMG}
+    entrypoint:
+    - tail
+    command: ["-f", "/dev/null"]
+    environment:
+      DB_ADDRESS: sqlite:////archive/engineering/test.db
+    volumes_from:
+    - BANZAIpipelineE2EData
+    - AstrometryCatalogs
+    labels:
+      io.rancher.container.pull_image: always
+      io.rancher.sidekicks: BANZAIpipelineE2EData,AstrometryCatalogs
+    logging:
+      options:
+        max-size: '100m'
+        max-file: '3'
+volumes:
+  banzaie2e:
+    driver: local
```

Comment:
I think it sets the driver to local by default.

Reply:
I believe I had to set this for Rancher or it wouldn't properly clean up the volume when the service was removed.